app-config.yaml

```diff
@@ -24,7 +24,6 @@ backend:
     csp:
         connect-src: [ "'self'", 'http:', 'https:' ]
         img-src: [ "'self'", "*.gov.bc.ca", "data:" ]
-        script-src: [ "'self'", "*.gov.bc.ca" ]
```

Comment:
This line was originally added when I added the snowplow js script to the header in the summer time.

I removed this line because it was interfering with the isolated-vm change. With this line in place, the website would show a blank page with the error in the web browser developer tools of:
"EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self' *.gov.bc.ca"."

I checked that the current analytics is working without it by using the Chrome snowplow plugin. I could see the page view for the environment I was working in (tools).